MatrixSDK/Data/Store/MXStore.h

```diff
@@ -31,13 +31,21 @@
 #import "MXRoomSummaryStore.h"
 
 @class MXSpaceGraphData;
+@class MXStoreService;
 
 /**
  The `MXStore` protocol defines an interface that must be implemented in order to store
  Matrix data handled during a `MXSession`.
  */
 @protocol MXStore <NSObject, MXRoomSummaryStore>
 
+#pragma mark - Store Management
+
+/**
+ The store service that is managing this store.
+ */
+@property (nonatomic, weak, nullable) MXStoreService *storeService;
```

Comment:
It can be interesting to remove this dependency. Probably in another PR. `MXStoreService` will orchestrate all data belong to a user from the outside. 
Maybe we should add callback or local notification on the `MXStore` that `MXStoreService` listen to.

Reply:
Agreed.